src/index.d.ts

```diff
@@ -57,6 +59,22 @@ declare namespace SDKStandardComponents {
         extensionList?: TExtensionList
     }
 
+    type TCredential = {
+        id: string | null;
```

Comment:
```suggestion
        id: string;
```
when `id` could be `null`

Reply:
It can't. fixed.